src/Bundler.js

```diff
@@ -332,26 +333,42 @@ class Bundler extends EventEmitter {
       let thrown = err;
 
       if (thrown.message.indexOf(`Cannot find module '${dep.name}'`) === 0) {
-        if (dep.optional) {
-          return;
-        }
-
-        thrown.message = `Cannot resolve dependency '${dep.name}'`;
-
-        // Add absolute path to the error message if the dependency specifies a relative path
-        if (dep.name.startsWith('.')) {
-          const absPath = Path.resolve(Path.dirname(asset.name), dep.name);
-          err.message += ` at '${absPath}'`;
-        }
-
-        // Generate a code frame where the dependency was used
-        if (dep.loc) {
-          await asset.loadIfNeeded();
-          thrown.loc = dep.loc;
-          thrown = asset.generateErrorMessage(thrown);
+        let isLocalFile = dep.name.startsWith('.');
+
+        if (!isLocalFile) {
+          try {
+            try {
+              logger.status(emoji.progress, `Installing ${dep.name}...`);
+              let dir = Path.dirname(asset.name);
+              await installPackage(dir, [dep.name], true, false);
+            } catch (npmError) {
+              logger.error(npmError.message);
```

Comment:
If install fails, should we continue or should it bail out? If the build would fail, could just bail out. If that's the case, could probably consolidate into a single `try...catch` .